scripts/kubernetes/meta-standalone.yaml

```diff
@@ -21,20 +21,11 @@ spec:
           - "meta-service.databend-system.svc.cluster.local"
       containers:
         - name: databend-meta
-          image: datafuselabs/databend-meta:v0.6.97-nightly
-          command: ["/bin/bash", "-c"]
+          image: datafuselabs/databend-meta:v0.6.100-nightly
```

Comment:
I think distroless image has different name? it is normal image

Reply:
we could wait until tomorrow for the new nightly image

Reply:
LGTM